### PR TITLE
[Enhancement] #212 add more explanation to the "delete record" confirma…

### LINKED ISSUE #212: [Enhancement] add more explanation to the "delete record" confirmation

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #212

### CODE REVIEW

**Comment by JavaLavaMT** [comment]
I understand about 1/2 of this

File: app/templates/components/confirm-modal.hbs [hunk 1]
```diff
@@ -8,7 +8,10 @@
     {{if title title 'Heads up!'}}
   </div>
   <div class="content">
-    {{if confirmContent confirmContent 'Are you sure you want to do that?'}}
+    {{if confirmContent confirmContent 'Are you sure you want to remove'}}
```
Comment (James1x0) [inline app/templates/components/confirm-modal.hbs]: @JavaLavaMT I think this needs to be
```
{{if confirmContent confirmContent (if deleteContext 'Are you sure you want to remove' 'Are you sure you want to do that?')}}
```

File: app/templates/components/confirm-modal.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@
     {{if title title 'Heads up!'}}
   </div>
   <div class="content">
-    {{if confirmContent confirmContent 'Are you sure you want to do that?'}}
+    {{if confirmContent confirmContent (if deleteContext (concat 'Are you sure you want to remove ' deleteContext '?') 'Are you sure you want to remove that?')}}
```
Comment (James1x0) [inline app/templates/components/confirm-modal.hbs]: @JavaLavaMT I still don't this is correct... As a default, the modal needs to say "Are you sure you want to do that?", if you have a deleteContext, the default is "Are you sure you want to remove `deleteContext`". If you have a confirmContent, you use that.

Comment (JavaLavaMT) [reply app/templates/components/confirm-modal.hbs]: I am sorry, I am not understanding, I have it set up if confirmContent then show confirmContent if not then see if deleteContext  then show deleteContex if no deleteContext then show 'Are you sure you want to remove that?'. can you help me understand

**Pull request merged**